config/common.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+/** @var array $params */
+
+use Yiisoft\Validator\RuleHandlerResolverInterface;
+use Yiisoft\Validator\Rules\Container\RuleHandlerContainer;
+
+return [
+    RuleHandlerResolverInterface::class => [
```

Comment:
Validator package will have default implementation of `RuleHandlerResolverInterface` and it will be used in configuration of `yiisoft/validator`. If we add configuration also here, then will be conflict.

We can add params, but don't add definition. Then add instruction about definition to documentation.

Reply:
`vendor-override-layer` config can't help in this case?

Reply:
> `vendor-override-layer` config can't help in this case?

It's will help, but this requires actions from user (add package to `vendor-override-layer` in app). If user install validator + this container, then by default he will get a configuration error.

Reply:
Removed and documented.